src/video/damage_tracker.h

```diff
@@ -5,14 +5,14 @@
 #include <stdbool.h>
 
 typedef struct damage_tracker {
-    vga_index dirty_range_start;
-    vga_index dirty_range_end;
+    vga_index dirty_range_first;
+    vga_index dirty_range_last;
     bool dirty;
 } damage_tracker;
 
 void damage_reset(damage_tracker *tracker);
 void damage_copy(damage_tracker *dst, const damage_tracker *src);
 void damage_combine(damage_tracker *dst, const damage_tracker *src);
-void damage_set_range(damage_tracker *tracker, vga_index start, vga_index end);
+void damage_dirty_range(damage_tracker *tracker, vga_index first, vga_index last); // dirties a range
```

Comment:
The struct is called damage; there is no special dirty range in damage because all damaged ranges are dirty ranges.

Reply:
In this, damage is a noun, which we are operating on. I could agree to something like damage_add_range() or somesuch.

Reply:
i'm amenable to damage_add_range, will rename it to that